packages/keyring-controller/src/KeyringController.ts

```diff
@@ -967,6 +985,21 @@ export class KeyringController extends BaseControllerV2<
       `${name}:getEncryptionPublicKey`,
       this.getEncryptionPublicKey.bind(this),
     );
+
+    this.messagingSystem.registerActionHandler(
+      `${name}:getAccounts`,
+      this.getAccounts.bind(this),
+    );
+
+    this.messagingSystem.registerActionHandler(
+      `${name}:getKeyringsByType`,
+      this.getKeyringsByType.bind(this),
+    );
+
+    this.messagingSystem.registerActionHandler(
+      `${name}:getKeyringForAccount`,
+      this.getKeyringForAccount.bind(this),
+    );
```

Comment:
Maybe would be better to do this in a different PR, so that we can review it separately (and potentially faster) - but I guess it was convenient for now to release preview packages in this PR to be tested in the extension

Perhaps we can also add some test cases to be sure that these actions work as expected: it should be fairly easy and we can take inspiration from all other actions test cases:
https://github.com/MetaMask/core/blob/515504115d3b70d3614c5f4684bba95b4c3fdea1/packages/keyring-controller/src/KeyringController.test.ts#L1908

Reply:
tests added in 369440a921ea9fe6a11cbf9e75f6dab922b5508b